modules/Quiz/resources/css/quiz.css

```diff
@@ -737,6 +737,13 @@ ol.second-row{
+    transition: width 0.3s,height 0.3s,top 0.3s,margin 0.3s ease-in-out;
+}
+.quizDone-cont.small{
+    width: 3vw;
+    height: 4vw;
+    top: 89vh;
+    margin-left: 97%;
```

Comment:
@MaksymKorniiov  
Have we reviewed this on fullscreen? How does 97% look in fullscreen mode?